Dockerfile

```diff
@@ -0,0 +1,25 @@
+# Based on example: 
+FROM node:20-slim AS base
+ENV PNPM_HOME="/pnpm"
+ENV PATH="$PNPM_HOME:$PATH"
+RUN corepack enable
+
+FROM base AS build
+COPY . /usr/src/app
+WORKDIR /usr/src/app
+RUN --mount=type=cache,id=pnpm,target=/pnpm/store pnpm install --frozen-lockfile
+RUN pnpm run -r build
+RUN pnpm deploy --filter=./apps/api --prod /prod/api
+
+# RUN pnpm deploy --filter=app2 --prod /prod/app2
```

Comment:
Should these comments stay?

Reply:
mmmm, yes imo. It might be helpful for anyone that may want to build an image for a new app.